solr/core/src/java/org/apache/solr/logging/LogWatcher.java

```diff
@@ -150,7 +149,7 @@ private static LogWatcher<?> createWatcher(LogWatcherConfig config, SolrResource
     String slf4jImpl;
 
     try {
-      slf4jImpl = StaticLoggerBinder.getSingleton().getLoggerFactoryClassStr();
+      slf4jImpl = LoggerFactory.getILoggerFactory().getClass().toString().replace("class ", "");
```

Comment:
```suggestion
      slf4jImpl = LoggerFactory.getILoggerFactory().getClass().getName();
```

Reply:
I was like this was dumb last night but couldn't figure out what I was missing. Thanks!